docs/src/free_associative_algebra.md

```diff
@@ -33,7 +33,7 @@ characteristic(R::FreeAssAlgebra)
 ## Free algebra constructors
 
 ```julia
-FreeAssociativeAlgebra(R::Ring, s::Vector{<: Union{String, Char, Symbol}}; cached::Bool = true)
+FreeAssociativeAlgebra(R::Ring, s::Union{Vector{String}, Vector{Char}, Vector{Symbol}}; cached::Bool = true)
```

Comment:
That's a good goal! The way I understand Bill is that he's suggesting something like this (where I also generalized `String` to `AbstractString`)
```suggestion
FreeAssociativeAlgebra(R::Ring, s::Union{AbstractVector{<:AbstractString}, AbstractVector{Char}, AbstractVector{Symbol}}; cached::Bool = true)
```
This would allow for other kinds of strings (string slices) and other vector implementations.

Reply:
Ok. I change it to this in the docs and the code.